ingest-asset-opex-creator/src/main/scala/uk/gov/nationalarchives/ingestassetopexcreator/XMLCreator.scala

```diff
@@ -163,7 +163,15 @@ class XMLCreator(ingestDateTime: OffsetDateTime) {
           }
       }
   </XIP>
-    IO.pure(xip.toString + "\n")
+    IO.pure {
+      <XIP xmlns="http://preservica.com/XIP/v7.0">
```

Comment:
How come you're doing this?

Reply:
The elements have to be in the correct order for the xsd validation to work. Because we're doing a map over the children, it ends up like:
```xml
<InformationObject/>
<Representation/>
<ContentObject/>
<Generation/>
<Bitstream/>
<ContentObject/>
<Generation/>
<Bitstream/>
```
This won't validate as it's in the wrong order so this line puts it back in the right order without having to map over the same collections multiple times.